### ISSUE TITLE
Caesar Cipher Encoder and Decoder in C++

### ISSUE THREAD

**Issue by kn1-gh7** [sherigar/HacktoberFest-2023#203]
Hi @sherigar. I want to create a Caesar Cipher encoder and decoder in c++. Can you please assign me this issue under hacktoberfest?

**Comment by Sgvkamalakar** [comment]
Certainly, I'd like to request the opportunity to address this challenge using Python. @sherigar

**Comment by SpiderMath** [comment]
I'd like to request the chance to do this in JavaScript

**Comment by GauraviVeling** [comment]
Hello. Please can you assign this issue to me. I would like contributing to this issue.
